pkg/entity/switch.go

```diff
@@ -101,22 +101,14 @@ func (w *ReplicationSwitchDowntimeOpts) GetMaxDuration() (time.Duration, bool) {
 
 // Contains all information about replication switch including its configuration and current status.
 type ReplicationSwitchInfo struct {
-	// Options for downtime switch
+	CreatedAt     time.Time  `redis:"createdAt"`
```

Comment:
Is it possible to keep comments?

Reply:
There is a couple of tools that are addressing this issue and next one as well.
I'll look into that.